src/fides/data/sample_project/fides.toml

```diff
@@ -14,12 +14,12 @@ db_index = 0
 
 [security]
 app_encryption_key = "examplevalidprojectencryptionkey"
-cors_origins = [ "http://localhost:8080", "http://localhost:3000",]
+cors_origins = [ "http://localhost:8080", "http://localhost:3001",]
 oauth_root_client_id = "fidesadmin"
 oauth_root_client_secret = "fidesadminsecret"
 
 [root_user]
-analytics_opt_out = false
+analytics_opt_out = true
```

Comment:
This one looks erroneous, pretty sure this can only be specified under `[user]`, below is a find in project of where the var is used:

![Screenshot 2022-10-30 at 16 03 03](https://user-images.githubusercontent.com/1667340/198899183-a00bbe74-43fc-4328-97b9-566e17431a45.png)

Reply:
Removed this, good eye